packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -3217,6 +3219,16 @@ class RenderSemanticsAnnotations extends RenderProxyBox {
     markNeedsSemanticsUpdate();
   }
 
+  /// If non-null, sets the [SemanticsNode.route] semantic to the give value.
+  bool get route => _route;
```

Comment:
"route", as a property name, sounds like it contains a route. This contains a bool, which doesn't match the name.

cc @goderbauer who ran into this problem with "password" recently and may have advice.

Reply:
I kept it this way for consistency with the other flags on `Semantics`, but I would have no problem renaming all of them to `isRoute`, `isPassword`, et cetera.  I could start with route in this CL and hit the other properties in a follow up, since that would be a breaking changed

Reply:
maybe `distinct`

Reply:
I really think we have to bite the bullet and rename these. A "routeName" can't be a bool. That's just weird.

Reply:
could be `namesRoute`?

Reply:
and `scopesRoute` for the other?

Reply:
cc @goderbauer

Reply:
Suggestions from @Hixie sound good to me.

Reply:
Done